sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -85,203 +85,267 @@ client = FormRecognizerClient(endpoint, credential)
 ## Key concepts
 
 ### FormRecognizerClient
-A `FormRecognizerClient` is the Form Recognizer interface to use for analyzing receipts and extracting layout items from 
-forms. It provides a different method based on input from a URL or input from a stream. 
-Use the receipt methods to extract receipt field values and locations from receipts from the United States from a URL
-or stream. Use the layout methods to extract table data and geometry from a URL or stream.
+A `FormRecognizerClient` is the Form Recognizer interface to use for recognizing data from forms using custom trained models,
+recognizing content and layout from forms, and analyzing receipts. It provides different methods
+based on inputs from a URL and inputs from a stream.
 
-### CustomFormClient
-A `CustomFormClient` is the Form Recognizer interface to use for creating, using, and managing custom machine-learned models. 
-It provides operations for training models on forms you provide, and extracting field values and locations from your 
-custom forms.  It also provides operations for viewing and deleting models, as well as understanding how close you 
-are to reaching subscription limits for the number of models you can train.
+#### Recognizing Custom Forms
+Our endpoint for recognizing custom forms takes in the ID of a custom model. The model can be trained either using this [library](#training-models "Training models")
```

Comment:
It's probably better to call these methods, not endpoints, as the methods don't always correspond directly to the REST endpoints.